frontend/pages/DashboardPage/cards/MDM/MDM.tsx

```diff
@@ -57,9 +57,9 @@ const EmptyMdmStatus = (): JSX.Element => (
       <>
         To see MDM versions, deploy&nbsp;
         <CustomLink
-          url="https://fleetdm.com/docs/using-fleet/adding-hosts#osquery-installer"
+          url="https://fleetdm.com/learn-more-about/fleetd"
           newTab
-          text="Fleet's osquery installer"
+          text="Fleet's agent (fleetd)"
```

Comment:
Taking a step back, I think we built this state to handle users/customers that deploy plain osquery.

The tables used to get MDM version info is only included in fleetd.

Now, I don't think we want to maintain code for users/customers that deploy plain osquery. Using fleetd is the best practice. 

That said, we don't want Fleet to break/be unusable if someone uses plain osquery.

If this is the case, what could we update this empty state to say instead?